### ISSUE TITLE
New instance of xtuple live, database not appearing in postgresql management in Webmin

### ISSUE THREAD

**Issue by mperreault1980** [xtuple/xtuple-server#280]
I've installed a new instance of xtuple live (quickstart) on Ubuntu 14.4LTS with the documentation at this site. I was trying to figure out how to allow access from outside for my database to my home IP, when I realized that from within Webmin, the database I created from the installation script doesn't show up in the postgresql databases. Anyone know why this may be?

Thx
MP

**Comment by cdatgnp** [comment]
I'm also experiencing this. From webmin, I can see only three databases:
postgres - when selected, webmin reports "This database has no tables, indexes, views or sequences."
template0 - for which webmin reports "This database is not currently accepting connections, so no actions can be performed in it."
template1 - which gives the same message as the 'postgres' database

When I connect to my database with pgAdmin3, I can see three databases on my server:
empty_live - which appears to have the xtuple system as I remember from a few years ago
gnp - which shares the name of the Ubuntu user I had invoke the `xtuple-server install-live` command
postgres - ... which really doesn't have any tables, indexes, views, or sequences in its only schema, 'public'

I hope the above helps. I'm going to continue to investigate...

**Comment by xtpclark** [comment]
That's correct - the default webmin postgres module is only configured to read port 5432, which I believe is always there as the pg_cluster named 'main'. Currently, xtuple-server does not configure any tools other than what we have under 'Other Commands'.
